src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -16,6 +16,19 @@
     <TestPath Condition="'$(TestPath)'==''">$(OutDir)</TestPath>
   </PropertyGroup>
 
+  <!--
+    Copied from Microsoft.Net.Test.Sdk.targets which currently skips netstandard projects as 
+    they are not intended to be executable. We override this as we launch the test assemblies
+    directly with the runner. This is needed for VS debugging support.
+    TODO: Fix VB test project reference to Test SDK.
+  -->
+  <PropertyGroup Condition="'$(IncludeVSTestReferences)' == 'true' AND '$(BuildingNETCoreAppVertical)' == 'true' AND '$(Language)' != 'VB'">
```

Comment:
`BuildingNETCoreAppVertical` is a very `corefx` specific concept. Does it really belong in buildtools? It definitely shouldn't go into arcade.

/cc @weshaggard

Reply:
right now the whole tests.targets file is mainly used by corefx and wcf. It's also imported by the coreclr tests but I'm not sure to what extent they are using it. As we planned to refactor the test framework and move it to arcade I would keep it in mind as a follow-up item.

Reply:
I agree with @ViktorHofer that this is pretty specific to corefx testing and would take a while to clean-up this for a general repo to use. There are other such usages in this targets file already. 

For the time being when we move this to arcade it will be corefx specific for a while until we have some other team interested in using it then we can generalize it.